providers/vmw_pvrdma/pvrdma_main.c

```diff
@@ -115,6 +115,7 @@ static int pvrdma_init_context_shared(struct pvrdma_context *context,
 	struct user_pvrdma_alloc_ucontext_resp resp;
```

Comment:
Change struct user_pvrdma_alloc_ucontext_resp resp;
to be struct user_pvrdma_alloc_ucontext_resp resp = {};

Reply:
Fixed